### PR TITLE
Content: Add new content for the new structure.

### PR DESCRIPTION
**Pull request by gabrielmocanu**
Signed-off-by: Viorel-Gabriel MOCANU (94548) <viorel.mocanu@stud.acs.upb.ro>

### CODE REVIEW

File: 04-injection-part-1/README.md [hunk 1]
```diff
@@ -434,22 +375,59 @@ Most instances of SQL injection can be prevented by using parameterized queries
...
 
-Parameterized queries can be used for any situation where untrusted input appears as data within the query, including the WHERE clause and values in an INSERT or UPDATE statement. They can't be used to handle untrusted input in other parts of the query, such as table or column names, or the ORDER BY clause. Application functionality that places untrusted data into those parts of the query will need to take a different approach, such as white-listing permitted input values, or using different logic to deliver the required behavior.
+# Further Reading
 
-For a parameterized query to be effective in preventing SQL injection, the string that is used in the query must always be a hard-coded constant, and must never contain any variable data from any origin. Do not be tempted to decide case-by-case whether an item of data is trusted, and continue using string concatenation within the query for cases that are considered safe. It is all too easy to make mistakes about the possible origin of data, or for changes in other code to violate assumptions about what data is tainted.
+* \[1] https://portswigger.net/burp
```
Comment (razvand) [inline 04-injection-part-1/README.md:407]: Why is there a backslash?

File: 04-injection-part-1/README.md [hunk 2]
```diff
@@ -434,22 +375,59 @@ Most instances of SQL injection can be prevented by using parameterized queries
...
+* [2] https://portswigger.net/support/configuring-your-browser-to-work-with-burp
+* [3] https://portswigger.net/support/checking-your-browser-proxy-configuration
+* [4] https://portswigger.net/support/installing-burp-suites-ca-certificate-in-your-browser
+* [5] https://portswigger.net/web-security/sql-injection/cheat-sheet
+* [6] https://github.com/swisskyrepo/PayloadsAllTheThings/tree/master/SQL%20Injection
+* \[7] https://sqlmap.org/
```
Comment (razvand) [inline 04-injection-part-1/README.md:413]: Why is there a backslash?

**Review by razvand — COMMENTED** [review]
I strongly support we deploy tutorial challenges with SQL injection attacks that we will use to demo the issues. And also students could test them. We provide all the explanations and solutions.

File: 04-injection-part-1/README.md (hunk 1 above)
Comment (gabrielmocanu) [reply 04-injection-part-1/README.md]: It was something to use hyperlink but now I delete them and use simple hyperlinks for Markdown.

File: 04-injection-part-1/README.md (hunk 2 above)
Comment (gabrielmocanu) [reply 04-injection-part-1/README.md]: It was something to use hyperlink but now I delete them and use simple hyperlinks for Markdown.

**Pull request merged**